CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Changelog
 
+## v1.2.0
+
+### Enhancements
+* move `verify-dockerfile` to `dockerfile verify` (#662)
```

Comment:
would be good to specifically call out breaking changes as breaking (see example in previous release)

Reply:
good call, updated!